cmd/mlpa/main.go

```diff
@@ -133,6 +133,7 @@ func run(ctx context.Context, logger *slog.Logger) error {
 	staticHash = url.QueryEscape(staticHash[3:11])
 
 	httpClient := &http.Client{Timeout: 10 * time.Second}
+	slowHTTPClient := &http.Client{Timeout: 30 * time.Second}
```

Comment:
I wouldn't mind us making this the default one, 10s is quite aggressive

Reply:
That simplifies things, I'll update 👍